### ISSUE TITLE
Useless back arrow

### ISSUE THREAD

**Issue by HaukeCornell** [fongandrew/hydeout#130]
The theme has a back arrow at the top of pages. E.g., on the [about page](https://fongandrew.github.io/hydeout/about). 

This back arrow link does not link back to the home page, but just refreshes the page. 
Should we remove it? How?

**Comment by frothedoatmilk** [comment]
I've just ran into the same issue. I believe there's some issue with [Jekyll's custom filters](https://jekyllrb.com/docs/liquid/filters/#find). A fix is to add your own `_includes/back-link.html` with the following:

```liquid
{% assign back_page = site.pages | where: "name", page.back_page | first %}
{% if back_page != null %}
    <p class="back-link">
    <a href="{{ back_page.url | relative_url }}"><span class="back-arrow icon">{% include svg/back-arrow.svg %}</span>{{ back_page.short_title | default: back_page.title }}</a>
    </p>
{% endif %}
```

**Comment by frothedoatmilk** [comment]
Yeah, so the `find` method was introduced in (this release)[https://github.com/jekyll/jekyll/releases/tag/v4.1.0], but GitHub pages is locked to 3.9.5 as of writing. So I'm not positive what's happening in this call when find doesn't exist. But it ends up returning the current page. Would be best to revert this behavior to `where` syntax until GH updates it's jekyll version?
